### PR TITLE
feat: adds lighthouse test and preview

### PR DESCRIPTION
**Pull request by bmuenzenmeyer**
### Check List

- [ ] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [ ] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing, and/or `npx turbo test:snapshot` to update snapshots if I created and/or updated React Components.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: .github/workflows/pull-request-target.yml [hunk 1]
```diff
@@ -242,3 +243,57 @@ jobs:
...
+  lighthouse-ci:
+    # We want to skip our lighthouse analysis on Dependabot PRs
+    if: startsWith(github.event.pull_request.head.ref, 'dependabot/') == false
+    name: Lighthouse Report
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Add comment to PR
+        # Signal that a lighthouse run is about to start
+        uses: marocchino/sticky-pull-request-comment@f61b6cf21ef2fcc468f4345cdfcc9bda741d2343
```
[the remark this answers is not in the document]
Comment (ovflowd) [reply .github/workflows/pull-request-target.yml:256]: BTW we already use this: `peter-evans/create-or-update-comment` which I assume does the same?

File: .github/workflows/pull-request-target.yml [hunk 2]
```diff
@@ -242,3 +243,57 @@ jobs:
...
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          number: ${{ github.event.issue.number }}
+          header: Lighthouse
+          message: |
+            Running Lighthouse audit...
+      - name: Capture Vercel preview URL
```
Comment (ovflowd) [inline .github/workflows/pull-request-target.yml:263]: We usually add blank lines between steps to ease readability!

File: .github/workflows/pull-request-target.yml [hunk 3]
```diff
@@ -242,3 +243,57 @@ jobs:
...
+    if: startsWith(github.event.pull_request.head.ref, 'dependabot/') == false
+    name: Lighthouse Report
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Add comment to PR
+        # Signal that a lighthouse run is about to start
+        uses: marocchino/sticky-pull-request-comment@f61b6cf21ef2fcc468f4345cdfcc9bda741d2343
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ovflowd) [inline .github/workflows/pull-request-target.yml:258]: I'd assume you don't need this parameter, this is probably implicitly passed.

File: .github/workflows/pull-request-target.yml [hunk 4]
```diff
@@ -242,3 +243,57 @@ jobs:
...
+          configPath: './lighthouserc.json'
+          urls: |
+            ${{ steps.vercel_preview_url.outputs.vercel_preview_url }}
+          uploadArtifacts: true
+      - name: Format lighthouse score
+        # Transform the audit results into a single, friendlier output
+        id: format_lighthouse_score
+        uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410
+        env:
+          LIGHTHOUSE_RESULT: ${{ steps.lighthouse_audit.outputs.manifest }}[0].summary
```
Comment (ovflowd) [inline .github/workflows/pull-request-target.yml:284]:
```suggestion
          LIGHTHOUSE_RESULT: ${{ steps.lighthouse_audit.outputs.manifest[0].summary }}
```

File: scripts/lighthouse/index.js
Comment (ovflowd) [inline scripts/lighthouse/index.js:1]: We use only `.mjs` with `use strict;`

**Comment by bmuenzenmeyer** [comment]
this has sufficiently changed that i am starting cleanly

**Pull request closed without merging**